examples/mongo/chart.test.ts

```diff
@@ -0,0 +1,29 @@
+import { MongoChart } from "./chart";
+import { Testing } from "cdk8s";
+import { TalisShortRegion, TalisDeploymentEnvironment } from "../../lib";
+
+describe("Mongo example", () => {
+  const PROCESS_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...PROCESS_ENV };
+    process.env.DOCKER_USERNAME = "someuser";
+    process.env.DOCKER_PASSWORD = "secret123";
```

Comment:
Same comment as in redis PR https://github.com/talis/talis-cdk8s-constructs/pull/18#discussion_r828916204